src/tests/functional/plugin/cpu/shared_tests_instances/behavior/ov_plugin/properties_tests.cpp

```diff
@@ -36,4 +36,45 @@ INSTANTIATE_TEST_SUITE_P(smoke_AutoMultiBehaviorTests, OVPropertiesTests,
                 ::testing::Values(CommonTestUtils::DEVICE_AUTO, CommonTestUtils::DEVICE_MULTI),
                 ::testing::ValuesIn(multi_Auto_properties)),
         OVPropertiesTests::getTestCaseName);
+
+const std::vector<ov::AnyMap> cpu_plugin_properties = {
+    {ov::hint::performance_mode(ov::hint::PerformanceMode::THROUGHPUT),
+     {CONFIG_KEY(EXCLUSIVE_ASYNC_REQUESTS), CONFIG_VALUE(YES)},
+     ov::hint::num_requests(2),
+     ov::enable_profiling(false)}};
+const std::vector<ov::AnyMap> cpu_compileModel_properties = {
+    {ov::hint::performance_mode(ov::hint::PerformanceMode::LATENCY),
+     {CONFIG_KEY(EXCLUSIVE_ASYNC_REQUESTS), CONFIG_VALUE(NO)},
+     ov::hint::num_requests(10),
+     ov::enable_profiling(true)}};
+
+INSTANTIATE_TEST_SUITE_P(smoke_cpuCompileModelBehaviorTests,
+                         OVSetPropComplieModleGetPropTests,
+                         ::testing::Combine(::testing::Values(CommonTestUtils::DEVICE_CPU),
+                                            ::testing::ValuesIn(cpu_plugin_properties),
+                                            ::testing::ValuesIn(cpu_compileModel_properties)),
+                         OVSetPropComplieModleGetPropTests::getTestCaseName);
+
+const std::vector<ov::AnyMap> auto_multi_plugin_properties = {
+    {ov::device::priorities(CommonTestUtils::DEVICE_CPU),
+     {CONFIG_KEY(EXCLUSIVE_ASYNC_REQUESTS), CONFIG_VALUE(YES)},
```

Comment:
Should we define this property in the properties.hpp like the definition of the other properties? @peterchen-intel 
`static constexpr Property<bool, PropertyMutability::RW> exclusive_async_requests{"EXCLUSIVE_ASYNC_REQUESTS"};`

Reply:
Currently, doesn't  plan to have this property in OV 2.0